tools/docker-compose/README.md

```diff
@@ -136,7 +186,7 @@ $ docker exec -ti tools_awx_1 awx-manage createsuperuser
 
 > Remember the username and password, as you will use them to log into the web interface for the first time.
 
-##### Load Demo Data
+##### Load demo data
```

Comment:
And this one.

Reply:
I lower-cased the headers at `#####` level and upper-cased the `###` (higher-level/larger) headers for consistency